lib/go/events/deposit.go

```diff
@@ -37,10 +38,18 @@ func (evt depositEvent) Owner() string {
 	return evt.To()
 }
 
+func (evt depositEvent) isValidEvent() bool {
```

Comment:
would a err response be more versatile than bool?

Reply:
there is an error here https://github.com/dapperlabs/nba-smart-contracts/pull/128/files#diff-c28b4f3f5872bedc52dcc2b8e8d09c8a0d1fa9a4b990cf12f1fd1ff06f9043c2R52 and since this is not an exported method, i don't think it is needed

Reply:
if we return an error here and rename to `validate() error`, then we could potentially surface more information about the error?
```go
fmt.Errorf("error decoding event: event is not a valid deposit event, expected type %s, got %s",expected,got)
```

Reply:
Done